src/styles/globalStyles.js

```diff
@@ -203,4 +203,29 @@ img {
   font-style: italic;
   vertical-align: middle;
 }
+input {
```

Comment:
@apsuhos you can add these styles inside the InputCls class

Reply:
Done